core/planner.c

```diff
@@ -757,7 +757,7 @@ bool plan(struct deco_state *ds, struct diveplan *diveplan, struct dive *dive, i
 
 	/* Find the first potential decostopdepth above current depth */
 	for (stopidx = 0; stopidx < decostoplevelcount; stopidx++)
-		if (*(decostoplevels + stopidx) >= depth)
```

Comment:
I can't comment on the patch itself. Just a coding-style comment:
Could we change the condition to `decostoplevels[stopidx] > depth`?
As it stands, that is super-cryptic.

Reply:
You are right. That code is ancient and was written before my time...